hack/make-rules/verify.sh

```diff
@@ -21,15 +21,10 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
 source "${KUBE_ROOT}/cluster/lib/util.sh"
 
-if [ -n "${VERBOSE}" ]; then
-    SILENT=false
-else
-    SILENT=true
-fi
-
 # Excluded checks are always skipped.
 EXCLUDED_CHECKS=(
   "verify-linkcheck.sh"  # runs in separate Jenkins job once per day due to high network usage
+  "verify-govet.sh"      # it has a separate make vet target
```

Comment:
If you look into [verify-govet.sh](https://github.com/kubernetes/kubernetes/blob/master/hack/verify-govet.sh) you'll notice the script will complain that it should not be called directly, but rather through `make vet`. So I've moved that call from here to `make verify` directly, which calls `hack/make-rules/vet.sh` which does not complain about not being called directly.